src/audio/lib/effects.c

```diff
@@ -104,13 +119,17 @@ s16 AudioEffects_GetVibratoPitchChange(VibratoState* vib) {
     s32 index;
 
     vib->time += (s32)vib->rate;
-    index = (vib->time >> 10) & 0x3F;
+    // 0x400 is 1 unit of time, 0x10000 is 1 period
+    index = (vib->time >> 10) % WAVE_SAMPLE_COUNT;
```

Comment:
```suggestion
    index = (vib->time / 0x400) % WAVE_SAMPLE_COUNT;
```
? 
Seems to work, and would help connect it to the comment